server/http_test.go

```diff
@@ -51,7 +51,7 @@ func (c *fakeConnector) TrustedEmailProvider() bool {
 
```

Comment:
They don't really get rejected in this case, it just changes the UI. Given that the real protection is below, I don't think it's crucial